libnixf/lib/Parse/test/Lexer.cpp

```diff
@@ -167,7 +162,7 @@ TEST_F(LexerTest, lexKW) {
   };
   auto Tokens = collect(Lexer, &Lexer::lex);
   for (size_t I = 0; I < sizeof(Match) / sizeof(TokenKind); I++) {
-    ASSERT_EQ(Tokens[I]->getKind(), Match[I]);
+    ASSERT_EQ(Tokens[I].Kind, Match[I]);
```

Comment:
this is somehow inconsistent between other types. Maybe we should write `getKind()`, I think it is more readable.